fastplotlib/graphics/_axes.py

```diff
@@ -243,7 +243,7 @@ class Axes:
     def __init__(
             self,
```

Comment:
would be nice to have a docstring here

Reply:
once it's more user-facing, we can do a second PR later to refactor axes with a more final version once we have an idea of exactly what we want